domain/src/main/java/org/oppia/android/domain/oppialogger/analytics/CpuPerformanceSnapshotterModule.kt

```diff
@@ -0,0 +1,29 @@
+package org.oppia.android.domain.oppialogger.analytics
+
+import dagger.Module
+import dagger.Provides
+import javax.inject.Qualifier
+
+private const val SIXTY_MINUTES_IN_MILLIS = 60 * 60 * 1000L
+private const val FIVE_MINUTES_IN_MILLIS = 5 * 60 * 1000L
```

Comment:
You could also return TimeUnit computed alternatives from the providers directly, rather than using constants. That might help a bit with readability.

Reply:
done.